README.md

```diff
@@ -51,7 +51,7 @@ Note that `keycloak-js` is a peer dependency of Keycloak Angular. This change al
 
 | Angular | keycloak-js | keycloak-angular |       Support       |
 | :-----: |:-----------:| :--------------: | :-----------------: |
-|  18.x   |   18 - 25   |      16.x.x      | New Features / Bugs |
+|  18.x   |   18 - 26   |      16.x.x      | New Features / Bugs |
```

Comment:
16.0.x will not respect this compatibility matrix update suggestion, only 16.1.x

Reply:
You're right, but I think it's the same for 15.x.x 🤔

Reply:
> You're right, but I think it's the same for 15.x.x 🤔

I believe it's better to correct the issue rather than repeat the same mistake just because it was done that way before. Let's add a new row for 16.1.x and Keycloak 18-26.x.x Once that's done, I’ll review and approve it (don't know if I can do that but I'll try), and then we can tag the maintainer for merging.

Reply:
Imho the table should be as simple as possible, one row per major angular version is easy to understand. Using 16.x.x (or ^16.0.0 using semantic versioning) npm should be able to fetch a working dependency version, namely 16.1.0 when using keycloak-js 26

Reply:
Ok, then this should be ready for merging @mauriciovigolo